docs/guides/admin/docs/configuration/load.md

```diff
@@ -57,21 +57,25 @@ is running.  The current files with relevant configuration keys are:
 | org.opencastproject.caption.impl.CaptionServiceImpl.cfg                                  | Caption convertion services        |
-| org.opencastproject.composer.impl.ComposerServiceImpl.cfg                                | Caption embedding services         |
-| org.opencastproject.distribution.distribution.streaming.StreamingDistributionService.cfg | Streaming distribution             |
+| org.opencastproject.distribution.aws.s3.AwsS3DistributionServiceImpl.cfg                 | AWS S3 distribution service        |
 | org.opencastproject.distribution.download.DownloadDistributionServiceImpl.cfg            | Download distribution              |
+| org.opencastproject.distribution.distribution.streaming.<br/>StreamingDistributionService.cfg | Streaming distribution             |
+| org.opencastproject.distribution.streaming.wowza.<br/>WowzaAdaptiveStreamingDistributionService.cfg | Adaptive streaming distribution |
 | org.opencastproject.execute.impl.ExecuteServiceImpl.cfg                                  | Execute service                    |
 | org.opencastproject.ingest.impl.IngestServiceImpl.cfg                                    | Ingest services                    |
 | org.opencastproject.inspection.ffmpeg.MediaInspectionServiceImpl.cfg                     | Media inspection using ffmpeg      |
-| org.opencastproject.inspection.impl.MediaInspectionServiceImpl.cfg                       | Media inspection using mediainfo   |
-| org.opencastproject.publication.youtube.YouTubePublicationServiceImpl.cfg                | Youtube distribution               |
 | org.opencastproject.publication.youtube.YouTubeV3PublicationServiceImpl.cfg              | Youtube distribution               |
 | org.opencastproject.search.impl.SearchServiceImpl.cfg                                    | Opencast engage index jobs         |
 | org.opencastproject.silencedetection.impl.SilenceDetectionServiceImpl.cfg                | Silence detection                  |
+| org.opencastproject.sox.impl.SoxServiceImpl.cfg                                          | Sox service                        |
 | org.opencastproject.textanalyzer.impl.TextAnalyzerServiceImpl.cfg                        | Text analysis, including slide OCR |
+| org.opencastproject.timelinepreviews.ffmpeg.TimelinePreviewsServiceImpl.cfg              | Timeline previews service          |
+| org.opencastproject.transcription.ibmwatson.IBMWatsonTranscriptionService.cfg            | IBM Watson start transcription job |
 | org.opencastproject.videoeditor.impl.VideoEditorServiceImpl.cfg                          | Video editor                       |
 | org.opencastproject.videosegmenter.ffmpeg.VideoSegmenterServiceImpl.cfg                  | Video segmentation                 |
+| org.opencastproject.waveform.ffmpeg.WaveformServiceImpl.cfg                              | Waveform generation for video editing |
```

Comment:
I'm wondering if we should just tell people to grab for `job.load`. I suspect this list may constantly be outdated.

Reply:
"Search the contents of the configuration directory (generally /etc/opencast) for `job.load` to find the relevant configuration keys"?

Seems reasonable to me.  We can already see how outdated this is...

Reply:
I agree and will make the change.